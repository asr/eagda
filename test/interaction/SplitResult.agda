{-# OPTIONS --copatterns #-}

module SplitResult where

open import Common.Product

test : {A B : Set} (a : A) (b : B) → A × B
test a b = {!!}

testFun : {A B : Set} (a : A) (b : B) → A × B
testFun = {!!}

