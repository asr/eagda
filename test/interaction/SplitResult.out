Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking SplitResult (SplitResult.agda).\n" t)
(agda2-info-action "*Type-checking*" " Checking Common.Product (Product.agda).\n" t)
(agda2-info-action "*Type-checking*" "  Checking Common.Level (Level.agda).\n" t)
(agda2-info-action "*Type-checking*" "  Finished Common.Level.\n" t)
(agda2-info-action "*Type-checking*" " Finished Common.Product.\n" t)
(agda2-info-action "*Type-checking*" "Finished SplitResult.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : .A × .B\n?1 : {A B : Set} → A → B → A × B\n" nil)
((last . 1) . (agda2-goals-action '(0 1)))
Agda2> ((last . 2) . (agda2-make-case-action '("proj₁ (test a b) = ?" "proj₂ (test a b) = ?")))
((last . 1) . (agda2-goals-action '(0 1)))
Agda2> ((last . 2) . (agda2-make-case-action '("proj₁ (testFun a b) = ?" "proj₂ (testFun a b) = ?")))
((last . 1) . (agda2-goals-action '(0 1)))
Agda2> Agda2> Agda2> Agda2> Agda2> Agda2> 